games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
```

Comment:
It might be a good idea to replace
```
	major = 0,
	middle = 0,
	minor = 0,
```
by
```
	major = 3,
	middle = 3,
	minor = 1,
```
but that's not very important.
BTW, in a few of our recipes, e.g. media-video/vlc/vlc-2.2.1.recipe or net-p2p/qbittorrent/qbittorrent-3.3.4.recipe, we have added a few extra steps to replace the hard-coded values (in major, middle and minor) by a pattern which we edit on the fly when we create a copy of rdef with these patterns replaced by the correct values.
It is not worth doing that for programs that do not get new releases, but it might be usefull the programs gets frequent updates.
I'm suggesting this in case other contributors think it could be a good idea.

~~Oops, wait, the game did not change since 2013, so using hard-coded values will be perfect. No need to use patterns. Sorry for the noise. (Keeping the message as you might find it usefull for other recipes ;-)~~

Hmm, Rocks'n'Diamonds 4.0.0.0 was released two days ago ;-)
So I guess it can be a good idea to use avoid hard-coded values.
This way, when we update the game, we won't need to edit the rdef file.

BTW, if you wish to update the game, you can do it in this task, or in a new task that we can create.